canonical-web-and-design/ubuntu.com
# Lower case c in all instances of "Ubuntu certified..."

## Done

- Replace uppercase C with lower case C in all instances of "Ubuntu certified..." for consistency 

## QA

- Checkout this branch
- Search codebase for "Ubuntu certified" and check all instances of capital C have been changed to lower case c


## Issue / Card

Fixes #10230
